x-pack/plugins/security_solution/public/common/translations.ts

```diff
@@ -7,6 +7,10 @@
 
 import { i18n } from '@kbn/i18n';
 
+export const SOLUTION_NAME = i18n.translate('xpack.securitySolution.pages.common.solutionName', {
+  defaultMessage: 'Security',
+});
+
 export const EMPTY_TITLE = i18n.translate('xpack.securitySolution.pages.common.emptyTitle', {
```

Comment:
I think this is dead now and can be removed.

Reply:
removed.